guides/code-style/.rubocop.yml

```diff
@@ -65,6 +57,13 @@ Style/PercentLiteralDelimiters:
     '%x': ()
 
 Style/StringLiterals:
-  Exclude:
-    - 'bin/*'
-    - 'db/schema.rb'
+  EnforcedStyle: single_quotes
+
+Lint/Debugger:
```

Comment:
Can we have these in alphabetical order?

Reply:
yes, sure i fix it:)